olp-cpp-sdk-dataservice-write/src/ApiClientLookup.cpp

```diff
@@ -126,6 +135,110 @@ client::CancellationToken ApiClientLookup::LookupApiClient(
       });
 }
 
+ApiClientLookup::ApiClientResponse ApiClientLookup::LookupApiClient(
```

Comment:
Please, make order of arguments same as in dataservice-read :
```
  static ApiClientResponse LookupApi(
      const client::HRN& catalog,
      client::CancellationContext cancellation_context, std::string service,
      std::string service_version, FetchOptions options,
      client::OlpClientSettings settings);
```

Reply:
We also need to +/- align other dataservice-write:: {{ApiClientLookup}} methods.